### PR TITLE
PRIME-2771 fix black screen issue

### PR DESCRIPTION
**Pull request by bergomi02**

### CODE REVIEW

File: prime-angular-frontend/src/app/shared/enums/college-licence-class.enum.ts [hunk 1]
```diff
@@ -4,5 +4,7 @@ export enum CollegeLicenceClassEnum {
   BCCNM = 3,
   CDSBC = 7,
   OptometryBC = 14,
-  OralHealth = 20
+  OralHealth = 20,
+  HealthCareProferssionals = 21,
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/shared/enums/college-licence-class.enum.ts:8]: Typo here

File: prime-angular-frontend/src/app/shared/enums/college-licence-grouping.enum.ts [hunk 2]
```diff
@@ -4,14 +4,52 @@ export enum CollegeLicenceGroupingEnum {
...
+  HealthProfessionOfOptomety = 15,
+  HealthProfessionOfPhysicalTherapy = 16,
+  HealthProfessionOfPsychology = 17,
+  HealthProfessionOfAudiology = 18,
+  HealthProfessionOfHearingInstrumentDispensing = 19,
+  HealthProfessionOfSpeechLanguagePathology = 20,
+  HealthProfessionOfChiropractic = 21,
+  HealthProfessionOfMassageTherapy = 22,
+  HealthProfessionOfNaturopathicMedicine = 23,
+  HealthProfessionOfTraditionalChineseMedicineAcupuncture = 24
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/shared/enums/college-licence-grouping.enum.ts:25]: Is this intended to be aligned with `CollegeLicenseGroupingConfiguration.cs`?  E.g. Dentist value differs 🤔

Comment (bergomi02) [reply prime-angular-frontend/src/app/shared/enums/college-licence-grouping.enum.ts:25]: What do you mean?
image

File: prime-angular-frontend/src/app/shared/enums/college-licence-class.enum.ts (hunk 1 above)
Comment (bergomi02) [reply prime-angular-frontend/src/app/shared/enums/college-licence-class.enum.ts:8]: it is fixed.

File: prime-angular-frontend/src/app/shared/enums/college-licence-grouping.enum.ts (hunk 2 above)
Comment (neophyte57) [reply prime-angular-frontend/src/app/shared/enums/college-licence-grouping.enum.ts:25]: Sorry, I was probably looking at `Weight` incorrectly.  That said, a couple of typos in `DentalTechnicain` and `DentalTerapist`

Comment (bergomi02) [reply prime-angular-frontend/src/app/shared/enums/college-licence-grouping.enum.ts]: Ops. it is fixed.

**Review by neophyte57 — COMMENTED** [review]
LGTM 👍

**Review by neophyte57 — APPROVED** [review]

**Pull request merged**